### ISSUE TITLE
Error de accesibilidad (color contrast) en vista principal de dashboard

### ISSUE THREAD

**Issue by StevenRey1** [FalofoAndes/MISW4103-GRUPO22#65]
## COMPORTAMIENTO ESPERADO:
Se debe garantizar que el contraste entre los colores de primer plano y de fondo cumpla con los umbrales mínimos de relación de contraste de WCAG 2 AA.

## COMPORTAMIENTO ACTUAL:
Los elementos:
.gh-dashboard-percentage, 
#gh-dashboard-anchor-date-start , ".gh-dashboard-engagement-30days > .gh-dashboard-metric > .gh-dashboard-metric-data > .gh-dashboard-metric-label.is-secondary",
.gh-dashboard-tab:nth-child(2) > .gh-dashboard-metric > .gh-dashboard-metric-data > .gh-dashboard-metric-label,
.is-large > .gh-dashboard-metric-data > .gh-dashboard-metric-label
no garantizan el contraste AA.

## PASOS PARA REPRODUCIR DEFECTO:
* Ingresar a ghost: https://ghost-ur1e.onrender.com/ghost/#/signin
* Ingresar las credenciales de administrador
* Iniciar el escaneo con el script de playwright creado por el grupo de automatizadores y verificar en la carpeta screenshots los issues con sus respectivas descripciones o hacer uso del plugin Axe instalado en google chrome.

## EVIDENCIAS:
![image](https://github.com/FalofoAndes/MISW4103-GRUPO22/assets/157659620/7cad0cc5-009e-44a4-9f8e-c13ab4ec7725)
## CONTEXTO:
En esta sección se debe detallar el contexto en el que fue ejecutado el escenario de prueba.

- Ghost versión: 5.82.0
- Sistema operativo. Windows 10
- Browser ( nombre y versión ): [Chrome 123.0.6312.88  ]

## INFORMACIÓN ADICIONAL:

Para resolver este problema, necesitas arreglar lo siguiente:

Element has insufficient color contrast of 3.49 (foreground color: #7c8b9a, background color: #ffffff, font size: 8.3pt (11px), font weight: normal). Expected contrast ratio of 4.5:1

helpUrl": https://dequeuniversity.com/rules/axe/4.9/landmark-unique?application=playwright
